interface/fax/fax_dispatch.php

```diff
@@ -614,203 +594,216 @@ function allCheckboxes(issel) {
+    <!-- From Section -->
+    <div class="form-group row">
+        <label class="col-2 col-form-label font-weight-bold"><?php echo xlt('From'); ?></label>
+        <div class="col-10">
+            <input type='text' size='10' name='form_from' class='form-control' data-toggle='tooltip' data-placement='top' title='Type your name here'>
+        </div>
+    </div>
+    <!-- To Section -->
+    <div class="form-group row">
+        <label class="col-2 col-form-label font-weight-bold"><?php echo xlt('To{{Destination}}'); ?></label>
+        <div class="col-10">
+            <input type='text' size='10' name='form_to' class='form-control' data-toggle='tooltip' data-placement='top' title='Type the recipient name here'>
+        </div>
+    </div>
+    <!-- Fax Section -->
+    <div class="form-group row">
+        <label class="col-2 col-form-label font-weight-bold"><?php echo xlt('Fax'); ?></label>
+        <div class="col-10">
+            <input type='text' size='10' name='form_fax' class='form-control' data-toggle='tooltip' data-placement='top' title='The fax phone number to send this to'>
+        </div>
+    </div>
+    <!-- Message Section -->
+    <div class="form-group row">
+        <label class="col-2 col-form-label font-weight-bold"><?php echo xlt('Message'); ?></label>
+        <div class="col-10">
+            <textarea name='form_message' rows='3' cols='30' class='form-control'
+                data-toggle='tooltip' data-placement='top'
+                title='Your comments to include with this message'>
+            </textarea>
+        </div>
+    </div>
+    <!-- Quality Section -->
+    <div class="form-group row">
+        <label class="col-2 col-form-label font-weight-bold"><?php echo xlt('Quality'); ?></label>
+        <div class="col-10">
+            <input type='radio' name='form_finemode' value=''><?php echo xlt('Normal'); ?>
+            <input type='radio' name='form_finemode' value='1' checked><?php echo xlt('Fine'); ?>
+        </div>
+    </div>
+</div>
+
+
+<p><span class="font-weight-bold"><?php echo xlt('Delete Pages'); ?>:</span>&nbsp;
 <input type='radio' name='form_cb_delete' value='2' />All&nbsp;
 <input type='radio' name='form_cb_delete' value='1' checked />Selected&nbsp;
 <input type='radio' name='form_cb_delete' value='0' />None
 </p>
 
-<center>
-<p>
-<input type='submit' name='form_save' value='<?php echo xla('OK'); ?>' />
-&nbsp; &nbsp;
-<input type='button' value='<?php echo xla('Cancel'); ?>' onclick='window.close()' />
-&nbsp; &nbsp;
-<input type='button' value='<?php echo xla('Select All'); ?>' onclick='allCheckboxes(true)' />
-&nbsp; &nbsp;
-<input type='button' value='<?php echo xla('Clear All'); ?>' onclick='allCheckboxes(false)' />
-</p>
 
-<p><br /><b><?php echo xlt('Please select the desired pages to copy or forward:'); ?></b></p>
+<div class="btn-group">
+    <button type='submit' class='btn btn-primary btn-save' name='form_save'><?php echo xla('OK'); ?></button>
```

Comment:
above will need to include the `value` setting to ensure it works correctly:
```
<button type='submit' class='btn btn-primary btn-save' name='form_save' value='<?php echo xla('OK'); ?>'><?php echo xla('OK'); ?></button>
```

Reply:
Updated the commit.